### ISSUE TITLE
Feature Request: Designate Protomech points

### ISSUE THREAD

**Issue by Xeno426** [MegaMek/megamek#951]
A problem I've come across that is rather aggravating is that it's extremely difficult to set custom protomech point sizes. If I want to make more than one under-strength protomech point, I have to add the number of points I want, and then delete the units to get them into separate points. Unfortunately, sometimes when I delete some protomech units the remaining ones reshuffle the point they're in, forcing me to delete those ones as well. This becomes particularly difficult when I am trying to field multiple mixed-protomech points.

For instance, I create three protomech points; the first one is 65-xx, the second is 66-xx, and the third is 67-xx. I delete one member from 65-xx, two from 66-xx, and two from 67-xx. However, when I check to see what point each protomech is in, I see a member of 66-xx has changed to be a member of 65-xx.

This also comes up when I am trying to set up Society protomech Uns, which have only three protomechs per Un.

tl;dr it would be nice if protomechs could be assigned to points rather like C3i networks, with a "network" limit of 5 units.

**Comment by Xeno426** [comment]
At the very least, it would be nice if point/group assignment of a protomech was a value saved in the unit export file. 

That way it could be manually edited and then imported.

**Comment by Dylan-M** [comment]
My goal here is to go by lance/star/point ID in MHQ if it exists, and also allow assigning it in MM.
That way "Lance" movement, moves the 'Mechs in *that* lance, rather than any 4 for example. And if there are 6 'Mechs in it (Comstar Level II for example), you move six.

**Comment by Xeno426** [comment]
How freeform would this be? Could you select any number of units for the lance/star/point/un/trey during setup? Would it allow mixing of organization types or for understrength units to be used?

**Comment by Dylan-M** [comment]
Basically yes. :P

**Comment by Xeno426** [comment]
Very cool. I look forward to it.

Don't forget Society vehicle Uns have 7 members to them.
